src/api/spec/features/webui/sign_up_spec.rb

```diff
@@ -6,13 +6,13 @@
   scenario "User" do
     visit root_path
 
-    fill_in 'login', with: user.login
-    fill_in 'email', with: user.email
+    fill_in 'login', with: 'eisendieter'
```

Comment:
I fear this will become a fundamental thing in the test suite when we're not finding a better way with VCR...